### ISSUE TITLE
Exception thrown on startup

### ISSUE THREAD

**Issue by boxxxie** [BrunoBonacci/ip-geoloc#6]
tested using version `0.2.3` and `snapshot`

seems like there is some issue with chesshire, maybe a newer version of it isn't compatible  with this library anymore.

```
Exception in thread "main" Syntax error macroexpanding at (cheshire/generate.clj:1:1).
...
...
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.core.JsonGenerationException
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:496)
	at java.base/java.lang.Class.forName(Class.java:475)
	at clojure.lang.RT.classForName(RT.java:2209)
	at clojure.lang.RT.classForNameNonLoading(RT.java:2222)
	at cheshire.generate$eval50428$loading__6789__auto____50429.invoke(generate.clj:1)
	at cheshire.generate$eval50428.invokeStatic(generate.clj:1)
	at cheshire.generate$eval50428.invoke(generate.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7194)
	at clojure.lang.Compiler.eval(Compiler.java:7183)
	at clojure.lang.Compiler.load(Compiler.java:7653)
	... 342 more
 ("exited abnormally with code 1")
```

**Comment by boxxxie** [comment]
i forked the repo, deleted the project.clj, made a deps.edn and updated all deps. things seem to be working smoothly now.

``` clojure
{:deps
 {
  clj-http/clj-http         {:mvn/version "3.12.3"}
  com.brunobonacci/safely   {:mvn/version "0.5.0"}
  com.maxmind.geoip2/geoip2 {:mvn/version "4.1.0"}
  org.clojure/clojure       {:mvn/version "1.11.1"}
  org.clojure/tools.logging {:mvn/version "1.2.4"}
  pandect/pandect           {:mvn/version "1.0.2"}
  org.clojure/core.cache    {:mvn/version "1.0.225"}
  }
 }

```

**Comment by BrunoBonacci** [comment]
This was probably due to the fact that `clj-http` stopped bundling `cheshire` as a dependency. 
anyway, i bumped all the dependencies and released as `0.3.1`.
